src/repositories/filterRepository.ts

```diff
@@ -0,0 +1,13 @@
+import { BodyType, IBodyTypeModel, IMarkModel, IModelModel, Mark, Model } from '../db/';
+
+export const getMarks = async (): Promise<IMarkModel[]> => {
+  return (await Mark.find()) as IMarkModel[];
```

Comment:
Do we force the types like this everywhere? Is it 100% needed?
If so let's keep it like this.